pom.xml

```diff
@@ -178,7 +192,7 @@
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
 			</plugin>
-			<plugin>
+<!-- 			<plugin>
```

Comment:
Build is failing currently with these dependencies commented out (will reinstate locally to complete review)